pkg/sql/codegen/pai/codegen_test.go

```diff
@@ -109,31 +109,10 @@ func mockClusterConfig() *ClusterConfig {
 }
 
 func mockSession() *pb.Session {
-	return &pb.Session{DbConnStr: "mysql://root:root@tcp(127.0.0.1:3306)/?maxAllowedPacket=0"}
-}
-
-func TestGetPAICmd(t *testing.T) {
-	a := assert.New(t)
-	cc := &ClusterConfig{
-		Worker: WorkerConfig{
-			Count: 1,
-			CPU:   2,
-			GPU:   0,
-		},
-		PS: PSConfig{
-			Count: 2,
-			CPU:   4,
-			GPU:   0,
-		},
+	return &pb.Session{
+		DbConnStr: "mysql://root:root@tcp(127.0.0.1:3306)/?maxAllowedPacket=0",
+		UserId:    "sqlflow",
```

Comment:
maybe we also need to cover the case when userid is `""`?

Reply:
Will do that in the future.